extensions/amp-position-observer/0.1/amp-position-observer.js

```diff
@@ -34,9 +34,10 @@ import {
 } from '../../../src/layout';
 import {
   installPositionObserverServiceForDoc,
+} from '../../../src/service/position-observer/position-observer-impl';
```

Comment:
Since the `position-observer-impl.js` `position-observer-entry.js` are much cleaner after the cleanup, I am OK if we want to move them to one single file. So that we don't need to import from different files or whitelist multiple files. 

Ideas?